### DISCUSSION TITLE
[Script request] duplicati

### DISCUSSION THREAD

**Discussion by CustomerJFK** [community-scripts/ProxmoxVE#923] [Request script] [answered]
### Application Name

duplicati

### Website

https://duplicati.com/

### Description

Hello
Can anyone be so kind to create a Proxmox LXC script for duplicati?

Thanks

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Please add description about this Tool.

**Comment by CustomerJFK** [comment]
Hi 
It's a self-hosted backup tool.
Thanks!

**Comment by tremor021** [comment, marked as answer]
Done here #2052
